src/core/SampleRecordHandle.cpp

```diff
@@ -111,7 +111,7 @@ f_cnt_t SampleRecordHandle::framesRecorded() const
 
 
 
-void SampleRecordHandle::createSampleBuffer( SampleBuffer** sampleBuf )
+void SampleRecordHandle::createSampleBuffer(SampleBuffer** sampleBuf)
```

Comment:
I am not too worried about both ``SampleRecordHandle`` and ``AudioSampleRecorder`` as it doesn't seem to be fully implemented. I don't want to change too much of this class as its kind of out of scope for me to do so. Any changes made here were really just to make the code compile (although any formatting changes can be undone here too).

Reply:
I think this is one of those changes that would actually make it easier to review. When reviewing `SampleClip`'s `setSampleBuffer()`, I had to find every place that called that method and verify that it was being passed an owning raw pointer instead of a non-owning raw pointer. But if `std::unique_ptr` was used instead, I could rely on the type system to do that for me, making the code both easier to understand and review.